resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -47,9 +47,19 @@ rm -f "$PIDFILE"
 
 init_restart_file "$restartfile" || exit $?
 
+CLASSPATH=${INSTALL_DIR}/puppet-server-release.jar
+
+if [ -e "${INSTALL_DIR}/cli/apps/cli-defaults.sh" ]; then
```

Comment:
I'd like to try to find a way to not have this file end up under the ../apps subdirectory so that the cli-app script doesn't try to treat the cli-defaults.sh script like another app, like for purposes of showing the available apps when the '-h' argument is passed in.  Not sure how easy that will be to do though?